packages/renderer-preact/src/index.js

```diff
@@ -13,10 +13,17 @@ export default (Base = HTMLElement) =>
       };
     }
     renderer(root, call) {
+      this._renderRoot = root;
       this._preactDom = render(
         call(),
         root,
         this._preactDom || root.childNodes[0]
       );
     }
+    disconnectedCallback() {
+      super.disconnectedCallback && super.disconnectedCallback();
+      // Preact hack https://github.com/developit/preact/issues/53
+      const Nothing = () => null;
```

Comment:
I believe Preact will happily accept `null` as the first argument to `render()` these days, the component should no longer be needed.

Here's an example: https://jsfiddle.net/developit/xngr7pm1/